news/2024/2024-03-24-4-digit-world-cup-2024-registrations-now-open.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "4 Digit World Cup 2024: Registrations Now Open!"
+date: 2024-03-24 12:00:00 +0000
+---
+
+Attention all 4 digit osu! players! The 4 Digit World Cup is back with its 2024 iteration — and a larger focus on competitiveness and player enjoyment!
+
+![](/wiki/shared/news/2024-03-24-4-digit-world-cup-2024-registrations-now-open/banner.png)
+
+The **4 Digit World Cup** (*pending forum link*), or 4WC for short, is an annual osu! world-cup-style tournament for players within the 4 digit (#1,000–9,999) rank range, featuring some of the best players of the scene battling it out on their biggest stage of the year for a chance to crown their country as the champions!
+
+[Last year](https://osu.ppy.sh/community/forums/topics/1761090) we had a staggering 53 countries competing, with Team United States barely claiming the first place finish on the [first Grand Finals set](https://osu.ppy.sh/community/matches/109632383), beating Team Canada on the tiebreaker after an excitingly close match featuring multiple back-to-back maps with microscopic score differences.
+
+## What's new?
+
+We are bringing in a number of small changes and improvements for this year to further emphasize and take advantage of the evolution of tournament play over the past year. We especially recommend returning players to read through these to stay up to date on what is going on:
+
+- **The qualifier stage will be featuring a larger (and slightly harder) mappool format**: it will now consist of 5 NoMod, 2 Hidden, 2 Hard Rock and 3 Double Time maps, reaching up to 7.1 stars of difficulty
+  - This change should help seed the teams, especially towards the top end, more accurately
+- **Teams will be provided a pre-made mappool for tryouts that they may choose to use**, intended to aid smaller teams in organising their tryouts
+- **There will be no more exceptions regarding the country players are playing for**
+  - In the past, flag changes had not been possible for a long time leading to exceptions for certain players and regions. However, as of [July 28th, 2023](https://osu.ppy.sh/home/news/2023-07-28-enabling-country-flag-changes), flag changes have been reinstated in an accessible way. Due to this change, you will now be able to play *only* for the country represented by your osu! profile
+- **Teams will now have one 2-minute timeout available per match** to use at their discretion. This timeout will be automatically applied by the referee would a team fail to pick, ban, or ready up in time
+- **The prizes have been raised up multiple notches**, incorporating a community-funded prize pool and various peripherals
```

Comment:
```suggestion
- **The qualifier stage will be featuring a larger (and slightly harder) mappool format**: it will now consist of 5 No Mod, 2 Hidden, 2 Hard Rock and 3 Double Time maps, reaching up to 7.1 stars of difficulty.
  - This change should help seed the teams, especially towards the top end, more accurately.
- **Teams will be provided a pre-made mappool for tryouts that they may choose to use**, intended to aid smaller teams in organising their tryouts.
- **There will be no more exceptions regarding the country players are playing for.**
  - In the past, flag changes had not been possible for a long time leading to exceptions for certain players and regions. However, as of [July 28th, 2023](https://osu.ppy.sh/home/news/2023-07-28-enabling-country-flag-changes), flag changes have been reinstated in an accessible way. Due to this change, you will now be able to play *only* for the country represented by your osu! profile.
- **Teams will now have one 2-minute timeout available per match** to use at their discretion. This timeout will be automatically applied by the referee would a team fail to pick, ban, or ready up in time.
- **The prizes have been raised up multiple notches**, incorporating a community-funded prize pool and various peripherals.
```